docs/libraries.md

```diff
@@ -17,12 +17,22 @@ This chapter will tell you how to make your library installable through Poetry.
 
 ## Versioning
 
-While Poetry does not enforce any convention regarding package versioning,
-it **strongly** recommends to follow [semantic versioning](https://semver.org).
-
-This has many advantages for the end users and allows them to set appropriate
+Poetry enforces [PEP 440](https://peps.python.org/pep-0440) as a version scheme for all projects. This
+is done so to be compliant with the larger Python ecosystem. For example, if Poetry builds a distribution
+for a project that uses a version that is not valid according to [PEP 440](https://peps.python.org/pep-0440),
+third party tools will be unable to parse the version correctly.
+
+Poetry does not enforce any versioning convention. Poetry does, however, encourage the use of
+[semantic versioning](https://semver.org/) as a convention within  scope of
+[PEP 440](https://peps.python.org/pep-0440). This has many advantages for the end users and allows them to set appropriate
 [version constraints]({{< relref "dependency-specification#version-constraints" >}}).
 
+The [PEP 440](https://peps.python.org/pep-0440) version scheme is not fully compatible with  all
+versioning schemes described by [semver](https://semver.org/). See
+[here](https://peps.python.org/pep-0440/#semantic-versioning) for more information. For example,
+`1.0.0-hotfix.1` is not compatible with [PEP 440](https://peps.python.org/pep-0440). You can instead
+choose to use `1.0.0-post1` or `1.0.0.post1`.
+
```

Comment:
I would make this section a warning and incorporate most of it into the above paragraph -- only put the concrete examples here.